packages/acceptance-tests/pkg-tests-specs/sources/commands/version/check.test.js

```diff
@@ -81,7 +81,7 @@ describe(`Commands`, () => {
       }),
     );
 
-    test(
+    test.skip(
       `it should detect that a dependent workspace changed vs master, even when the current workspace already has a bump scheduled`,
```

Comment:
I don't understand why it should fail. If we want to consider untracked changes then we should include an untracked deferred version bump as well. Or we ignore untracked changes which means we can't consider untracked version bumps (which would be tricky).

Reply:
> If we want to consider untracked changes then we should include an untracked deferred version bump as well

In this test case the version bump isn't untracked - it's been tracked as part of the commit on master at line 90.

Reply:
Sure but why would we throw here? There's a change and a bump happened. Everything is fine if these changes are committed. Everything is fine if no changes are committed. Everything is fine if only package.json is committed. Only when the change is committed but not the bump then we might have an issue.

This doesn't seem like a useful distinction to me.

Edit: Or is the test order just weird? Would you want to throw even without L95?

Reply:
Depends what you mean.

- If L95 was completely removed, I'd still throw but this time because `a` is undecided
- If L95 was turned into a `decline`, I wouldn't throw because the changes to `c` are supposed to be internal

But if `a` is bumped (as is the case here), then `c` must also be decided in the same PR. Even if master already encodes a bump information, it might not match what the current PR would require (for example, a previous PR might have been resolved with a `minor`, but our current one would require a `major`).